### ISSUE TITLE
VOD Request

### ISSUE THREAD

**Issue by Manescha** [TwitchRecover/TwitchRecover#305]
Hello id like to ask for a vod recover
The tool can find the urls but while trying to download the file i just creates countless 1kb files under %temp%
Hope you could help!

Vod in question:
https://twitchtracker.com/jadez/streams/46983646429

Url from tool:
https://d1m7jfoe9zdc1j.cloudfront.net/68498c6aab207d7b0d61_jadez_46983646429_1660433104/chunked/index-dvr.m3u8

Cheers

**Comment by ItIckeYd** [comment]
@Manescha 

Enter M3U8 Link: https://d1m7jfoe9zdc1j.cloudfront.net/68498c6aab207d7b0d61_jadez_46983646429_1660433104/chunked/index-dvr.m3u8
431 of 1232 segments are valid.

Not completely sure if this is the reason but it is my assumption that due to most of the segments being invalid it is causing issues with the downloading functionality of the application.

**Comment by Manescha** [comment]
@ItIckeYd 
Hey thank you for the response, its the same url ive posted but even in the best case only 1/3 of the vod would be playable?

**Comment by ItIckeYd** [comment]
@Manescha yes. Once the video files (.ts) which make up a m3u8 playlist are removed there is no way to get the full vod only parts where the segments (.ts) are available.


This happens a lot where the vod is retrievable but not playable because all the video files were removed hence why I added a check to count which segments are available vs not.
